ethereum/core/build.gradle

```diff
@@ -55,6 +55,8 @@ dependencies {
   integrationTestImplementation 'junit:junit'
   integrationTestImplementation 'org.assertj:assertj-core'
   integrationTestImplementation 'org.mockito:mockito-core'
+  integrationTestImplementation project(':testutil')
+  integrationTestImplementation 'net.consensys:orion:0.1.0-SNAPSHOT'
```

Comment:
All dependencies versions are being set in the `gradle/versions.gradle` file. Can we do the same with this dependency?